dev/maven-plugin/src/main/scala/com/lightbend/lagom/maven/ServerMojos.scala

```diff
@@ -152,12 +158,10 @@ class StartServiceLocatorMojo @Inject() (logger: MavenLoggerProxy, facade: Maven
 
       val scalaClassLoader = scalaClassLoaderManager.extractScalaClassLoader(cp)
 
-      val scalaUnmanagedServices =
-        if (cassandraEnabled) {
-          StaticServiceLocations.withCassandraLocation(cassandraPort, unmanagedServices.asScala.toMap)
-        } else {
-          unmanagedServices.asScala.toMap
-        }
+      val theKafkaAddress = if (kafkaAddress == null) s"127.0.0.1:$kafkaPort" else kafkaAddress
+
+      val scalaUnmanagedServices = StaticServiceLocations.staticServiceLocations(cassandraPort, theKafkaAddress) ++
```

Comment:
Yes. I don't think anything has changed with regards to Kafka, the `kafkaAddress` property has always allowed you to set it and pass it to the serivce in dev mode for when you use an external Kafka - this exact use case is in fact document.  What has changed is Cassandra, previously, if you disabled Cassandra, then it wouldn't be locatable in the service locator - now it's always available regardless of whether Cassandra is enabled or not, which allows you to use an external Cassandra.